cumulusci/salesforce_api/utils.py

```diff
@@ -12,12 +13,27 @@
 def get_simple_salesforce_connection(
     project_config, org_config, api_version=None, base_url: str = None
 ):
+
     # Retry on long-running metadeploy jobs
     retries = Retry(total=5, status_forcelist=(502, 503, 504), backoff_factor=0.3)
     adapter = HTTPAdapter(max_retries=retries)
+    instance_url = org_config.instance_url
+
+    # Attept to get the host and port from the URL, but ignore any errors retrieving it.
+    try:
+        instance_url = urlparse(org_config.instance_url).hostname
```

Comment:
I would call the variable `instance` here since it's no longer a full URL, and pass that along on line 20.